LUKeychainAccess/LUKeychainAccess.m

```diff
@@ -149,11 +149,21 @@ - (id)objectForKey:(NSString *)key {
   return object;
 }
 
+- (id)recursivelyFindObjectForKey:(NSString *)key fromBaseClass:(Class)cls {
```

Comment:
Not sure I agree with the naming here. Doesn't "base class" mean the class which other classes are derived? You could probably just say `fromClass` instead.